modules/apps/portal-search-solr8/portal-search-solr8-impl/src/main/java/com/liferay/portal/search/solr8/internal/search/engine/adapter/search/SolrSearchRequestExecutor.java

```diff
@@ -35,6 +37,13 @@
 )
 public class SolrSearchRequestExecutor implements SearchRequestExecutor {
 
+	@Override
+	public ClearScrollResponse executeSearchRequest(
+		ClearScrollRequest clearScrollRequest) {
+
+		return new ClearScrollResponse(false);
```

Comment:
just throw new UnsupportedOperationException(); instead

thats the no-op pattern for solr